src/pages/TokenDetails/index.tsx

```diff
@@ -94,12 +175,31 @@ export default function TokenDetails() {
             <BreadcrumbNavLink to={`/tokens/${chainName}`}>
               <ArrowLeft size={14} /> Tokens
             </BreadcrumbNavLink>
-            <ChartSection
-              token={tokenQueryData}
-              currency={token}
-              nativeCurrency={isNative ? nativeCurrency : undefined}
-              prices={prices}
-            />
+            <ChartHeader>
```

Comment:
This is already a large file - can ChartHeader be moved to `components/Tokens/TokenDetails/ChartHeader.tsx`? Alternatively, have this be `components/Tokens/TokenDetails/Chart.tsx` and nest `ChartSection.tsx` within this.